src/bmi_calculator/core.clj

```diff
@@ -5,11 +5,11 @@
             [orchestra.spec.test :as st])
   (:gen-class))
 
-(spec/def ::weight (spec/and double? (complement empty?)))
+(spec/def ::weight double?)
 (spec/def ::height double?)
 (spec/def ::bmi    (spec/and pos? double?))
 (spec/def ::present-string (spec/and string? (complement empty?)))
-(spec/def ::values-map map?)
+(spec/def ::values-map (spec/keys :req-un [::weight ::height]))
```

Comment:
`values-map` is not an optimal name

Why? b/c when I find it in code, I'll have no idea what it means

Better: `::person`